pages/vi/profiles/nachiketdhamankar.md

```diff
@@ -0,0 +1,13 @@
+# Nachiket Dhamankar
+
+:house: Boston, MA <br />
+:clock1: Eastern Time <br />
+:computer: Ubuntu 18.04 <br />
+
+Hello everyone! <br />
+I am currently a Computer Science student at Northeastern University, Boston. <br />
+In my free time I like playing badminton and watching movies and I think Batman is better than Superman. <br /> <br />
+
+:link: [Webpage](www.nachiket.me) <br />
```

Comment:
Link is broken.

Reply:
Noticed that the link would assume it's a relative path. Fixed it.
```suggestion
:link: [Webpage](www.nachiket.me) <br />
```